src/sentry/quotas/base.py

```diff
@@ -70,40 +62,54 @@ def get_project_quota(self, project):
         max_quota_share = int(OrganizationOption.objects.get_value(
             org, 'sentry:project-rate-limit', 100))
 
-        if not legacy_quota and max_quota_share == 100:
-            return 0
-
-        org_quota = self.get_organization_quota(org)
+        if max_quota_share == 100:
+            return (0, 60)
```

Comment:
If we're changing the return type here anyway, can we also change the "no quota" value to `None` instead of `0`? This  confuses me for a while each time I read through it and the only reason I left the previous behavior when I rewrote some of this a few months ago was because I didn't want to break any existing backends that were swapped in.